package.json

```diff
@@ -4,9 +4,13 @@
   "main": "index.js",
   "author": "Joshua Comeau",
   "scripts": {
-    "start": "serve ."
+    "start": "live-server ."
   },
   "dependencies": {
-    "serve": "^11.3.2"
+    "live-server": "^1.2.1"
+  },
+  "engines": {
+    "npm": "please-use-yarn",
```

Comment:
This change will prevent `npm install` from working (it has to be combined with the `engine-strict = true` setting in the `.npmrc`